android/build.gradle

```diff
@@ -14,6 +14,8 @@ buildscript {
     }
     dependencies {
         classpath("com.android.tools.build:gradle:4.2.2")
+        // Google services dependencies to be used by Firebase
+        classpath("com.google.gms:google-services:4.3.13")
```

Comment:
Is this an application dependency? Should we add it to the "individual module build.gradle files" as per the comment below? Or should this really be here?

Reply:
As the documentation state, it should be here.